src/api/HttpHelper.tsx

```diff
@@ -1,19 +1,19 @@
-import { ApiRequest, HttpApi, RequestType } from "./ApiTypes.d";
+import { ApiRequest, HttpApi } from "./ApiTypes.d";
 import { Base64 } from "js-base64";
 import { v4 as generateUUID } from 'uuid';
 import { toast } from "react-toastify";
 import cacheUtils from "../utils/CacheUtils";
 import { getProperty } from "../utils/PropertiesUtils";
+import { getNextMessageId } from "../utils/MessageIdUtils";
 
 const commandEndpoint = getProperty("commandEndpoint");
 let requests: ApiRequest[] = [];
 
 function sendRequest(request: ApiRequest): Promise<void> {
+    request.mId = getNextMessageId();
     let requestString = JSON.stringify(request.data);
     let headers = { 'ConId': getOrGenerateUUid() };
     var url = `${commandEndpoint}/${request.type}/${Base64.encode(requestString)}`;
-    if (request.mId)
-        url += `/${request.mId}`;
```

Comment:
the cloudflare cache has to be skipped if the command is valid for less than half an hour.
`sendRequestLimitCache` used the mId to for this but now can't